### PR TITLE
feat: Add `LocalSettings`-Sample

### PR DESCRIPTION
**Pull request by TopperDEL**
GitHub Issue (If applicable): #492

## PR Type

What kind of change does this PR introduce?

- Sample

## What is the current behavior?

There is no sample for `LocalSettings`.

## What is the new behavior?
There is now a sample for `LocalSettings`.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [X] Tested on Wasm.
- [X] Tested on Android.
- [X] Tested on UWP.
- [X] Tested in both **Light** and **Dark** themes.
- [X] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by TopperDEL** [comment]
I kept the logic the same like in #515

**Review by MartinZikmund — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	  xmlns:wasm="http://uno.ui/wasm"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  mc:Ignorable="d android ios macos skia wasm"
+	  x:Name="page">
+
+	<Page.Resources>
+		<ResourceDictionary>
+            <samples:ClipboardSamplePageViewModel x:Key="LocalSettingsSamplePageViewModel"/>
+		</ResourceDictionary>
+	</Page.Resources>
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml:22]: Is this required? Looks to be a view model for the Clipboard sample page

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs:1]: Looks like we got a mix of tab and space whitespacing, let's keep it all tabs

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.Features, "Local Settings", Description = "Use settings to remember the user-customizable aspects of your app. For example, a news reader could use app settings to save which news sources to display and what font to use for reading articles.", DocumentationLink = "https://learn.microsoft.com/en-us/windows/uwp/get-started/settings-learning-track")]
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs:15]: If https://github.com/unoplatform/Uno.Gallery/pull/514 merges first, we can change this to `SampleCategory.NonUIFeatures`

**Review by kazo0 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml (hunk 1 above)
Comment (TopperDEL) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml]: Yeah, I copied the page from the Clipboard sample. I thought about providing a way to provide settings by the user but switched to the fixed key.
I removed the ViewModel.

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs (hunk 2 above)
Comment (TopperDEL) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs:1]: Cleaned up!

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs (hunk 3 above)
Comment (TopperDEL) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs]: Merge master und switched to NonUIFeatures

**Comment by TopperDEL** [comment]
Should be all fine now! @kazo0

**Pull request merged**